Formula/hyfetch.rb

```diff
@@ -0,0 +1,24 @@
+class Hyfetch < Formula
+  desc "Maintained fork of Neofetch"
+  homepage "https://github.com/dylanaraps/neofetch"
+  url "https://github.com/hykilpikonna/hyfetch/archive/refs/tags/1.4.10.tar.gz"
+  sha256 "a8988b081c79a4a80d92262e3588fab1ad7f890ff6dbfb4ed76f697be1f8d903"
+  license "MIT"
+  head "https://github.com/dylanaraps/neofetch.git", branch: "master"
+
+  on_macos do
+    depends_on "screenresolution"
+  end
+
+  conflicts_with "neofetch", because: "conflicting name"
+
+  def install
+    inreplace "neofetch", "/usr/local", HOMEBREW_PREFIX
+    system "make", "install", "PREFIX=#{prefix}"
```

Comment:
Per comments from @chenrui333, you can just circumvent the Makefile to prevent any conflicts

```suggestion
    inreplace "neofetch", "/usr/local", HOMEBREW_PREFIX
    bin.install "neofetch" => "neowofetch"
    man1.install "neofetch.1" => "neowofetch.1"
```

Reply:
but `neofetch` users are anticipating `neofetch` binary? if that is the case, then the binary rename would hurt UX?

Reply:
~~Then the `conflicts_with` statement should be preserved~~ it has been